pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[tool.black]
+line-length = 119
+target-version = ['py38']
```

Comment:
Should we instead have:

`target-version = ['py37', 'py38', 'py39']`

As is done in sklearn: https://github.com/scikit-learn/scikit-learn/blob/main/pyproject.toml

Reply:
Thanks. Done.